frontend/web/components/ProjectFilter.tsx

```diff
@@ -2,7 +2,7 @@ import { FC, useMemo } from 'react'
 import { useGetProjectsQuery } from 'common/services/useProject'
 
 export type ProjectFilterType = {
-  organisationId: string
+  organisationId: string | number
```

Comment:
Can we really not support just `number` here?

Reply:
I presumed this component was used everywhere and didn't want to double the PR size, though looking at it, it turns out there aren't many. Adjusted